integration_test/test.js

```diff
@@ -0,0 +1,48 @@
+import { check } from 'k6'
+import http from 'k6/http'
+
+const UNLEASH_BASE_URL = __ENV.UNLEASH_BASE_URL
+const API_CLIENT_FEATURES_URL = `${UNLEASH_BASE_URL}/api/client/features`
+const CLIENT_ACCESS_TOKEN = __ENV.CLIENT_ACCESS_TOKEN
+
+export const options = {
+  thresholds: {
+    checks: ['rate==1']
+  },
+  scenarios: {
+    homepage: {
+      exec: 'homepage',
+      executor: 'per-vu-iterations',
+      iterations: 1,
+      vus: 1
+    },
+    apiClientFeatures: {
+      exec: 'apiClientFeatures',
+      executor: 'per-vu-iterations',
+      iterations: 1,
+      vus: 1
+    }
+  }
+}
+
+export function homepage () {
+  const res = http.get(UNLEASH_BASE_URL)
+
+  check(res, {
+    'homepage response code was 200': (res) => res.status === 200
+  })
+}
+
+export function apiClientFeatures () {
```

Comment:
What if you remove the token, does it still give 200?

Reply:
401, added one more test case in https://github.com/app-sre/unleash/pull/54/commits/729ca278f00c04840951f27a5842f04f0f6a04c3